target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4018-dlink-dap-2610.dts

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019-ap.dk01.1.dtsi"
```

Comment:
I would prefer if you could rebase this DTS on top of the qcom-ipq4019.dtsi and not the "qcom-ipq4019-ap.dk01.1.dtsi".
Reason being that "qcom-ipq4019-ap.dk01.1.dtsi" is part of qualcomm's EVA boards and out of our control (you could end up with unrelated changes in these files that break your device).

Reply:
This was some work, but I can see your point. I rebased the DTS, and did not enable the usb devices since they are not actually available on this hardware.